<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="Result">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>511</width>
      <height>341</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Montserrat&quot;;</string>
    </property>
    <property name="title">
     <string>Result</string>
    </property>
    <widget class="QLabel" name="ProfNameLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Montserrat&quot;;</string>
     </property>
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
    <widget class="QLabel" name="ProfName">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>40</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Montserrat&quot;;</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="ProfImage">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>20</y>
       <width>181</width>
       <height>171</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Montserrat&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="CurrStatusLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Montserrat&quot;;</string>
     </property>
     <property name="text">
      <string>Current Status:</string>
     </property>
    </widget>
    <widget class="QLabel" name="CurrStatus">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>90</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Montserrat&quot;;</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
